src/Projections/Reunion/Microsoft.UI.Dispatching.DispatcherQueueSynchronizationContext.cs

```diff
@@ -1,14 +1,17 @@
 using System;
 using System.Threading;
-using Microsoft.System;
+using WinRT;
+
+#nullable enable
+#pragma warning disable CA1416
```

Comment:
>CA1416 [](http://example.com/codeflow?start=24&length=6)

Could you add a comment documenting why this is disabled?

Reply:
Done in 52dd2ddb3a6a036bbb9d8f7f829bde9a85d89039 👍